lib/utils.js

```diff
@@ -81,3 +81,27 @@ export function makeTag(tagFormat, version) {
 export function isSameChannel(channel, otherChannel) {
   return channel === otherChannel || (!channel && !otherChannel);
 }
+
+export function getFilesToCommit(plugins) {
+  const files = new Set();
+
+  // Get files from changelog plugin
+  const changelogPlugin = plugins.find(plugin =>
+    Array.isArray(plugin) && plugin[0] === '@semantic-release/changelog');
+  if (changelogPlugin && changelogPlugin[1] && changelogPlugin[1].changelogFile) {
+    files.add(changelogPlugin[1].changelogFile);
+  }
+
+  // Get files from replace plugin
+  const replacePlugin = plugins.find(plugin =>
```

Comment:
How does this bit work, it seems to only collect file names

Reply:
It gets the files to bump from the config file. 
I tested it here: https://github.com/contiamo/tests-greg/blob/main/.releaserc.json#L37